profile.html

```diff
@@ -40,6 +40,7 @@
 <script type="module" src="./sidebar.js"></script>
 <script type="module" src="./actor-profile.js"></script>
 <script type="module" src="./post.js"></script>
+<script type="module" src="./p2p-media.js"></script>
```

Comment:
We should have post.js import any of it's dependencies directly instead of doing it top level.

Similarly outbox should import post.